### PR TITLE
Mac os install changes

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
List of changes:
Reorganized topic.
Added section on installing Grafana using standalone binaries.
Also added a "next steps" section.

### CODE REVIEW

File: docs/sources/installation/mac.md [hunk 1]
```diff
@@ -10,37 +10,61 @@ weight = 500
...
+To install a nightly build, or to install the latest version of Grafana  without Homebrew, go to the [Grafana download page](https://grafana.com/grafana/download/7.3.0-381ff45epre?platform=mac).
+
+1. Select the Grafana version you want to install. By default, the most recent released version is selected.
+
+   > **Note:** The downloads page lists only finished releases. If you want to install a beta version, click [Nightly ] **Nightly Builds** and then select a version.
 
-1. On the [Grafana download page](https://grafana.com/grafana/download?platform=mac), select the Grafana version you want to install. 
-   * The most recent Grafana version is selected by default.
-   * The **Version** field displays only finished releases. If you want to install a beta version, click **Nightly Builds** and then select a version.
 2. Select an **Edition**.
```
Comment (oddlittlebird) [inline docs/sources/installation/mac.md:47]: Please change this numbering to lazy numbering.

File: docs/sources/installation/mac.md [hunk 2]
```diff
@@ -10,37 +10,61 @@ weight = 500
...
    * **Enterprise** - Recommended download. Functionally identical to the open source version, but includes features you can unlock with a license if you so choose.
 3. Click **Mac**.
-4. Copy and paste the code from the installation page into your command line and run. It follows the pattern shown below.
-
+4. Open a terminal and download the binary using the cURL command. The following example shows Grafana 7.1.5 version:
    ```bash
-   brew update
-   brew install grafana
-   ```
+   curl -O https://dl.grafana.com/oss/release/grafana-7.1.5.darwin-amd64.tar.gz
```
Comment (oddlittlebird) [inline docs/sources/installation/mac.md:53]: These instructions do not match what is going on in the official download page: https://grafana.com/grafana/download?platform=mac

I do not want to repeat the downloads page. That's inefficient and annoying for us to maintain. 

Also, we don't want to diverge too much from the official downloads page. If it is wrong, then we need to talk to the website team about fixing it.

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

File: docs/sources/installation/mac.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/installation/mac.md]: Fixed.

File: docs/sources/installation/mac.md (hunk 2 above)
Comment (achatterjee-grafana) [reply docs/sources/installation/mac.md:53]: The download page should not show the brew steps.  That is install information, and moreover, the user can perform those steps from the Homebrew and not the Grafana downloads page.

The macOS section on the download page should be updated.

File: docs/sources/installation/mac.md [hunk 3]
```diff
@@ -10,37 +10,61 @@ weight = 500
 
 # Install on macOS
 
-This page provides instructions to help you install Grafana on macOS. 
+This page explains how to install Grafana and get the service running on your macOS. It has the following topics:
```
Comment (oddlittlebird) [inline docs/sources/installation/mac.md:13]: We don't generally use the mini-TOC. It's redundant with the auto-generated side menu. Please remove.

File: docs/sources/installation/mac.md (hunk 2 above)
Comment (oddlittlebird) [reply docs/sources/installation/mac.md:53]: @DanCech, let's chat about this on Monday.

**Review by oddlittlebird — APPROVED** [review]
One minor edit to maintain consistency with all the other pages and avoid redundancy. Otherwise, good to go.

**Pull request merged**